controllers/map-filter.js

```diff
@@ -35,8 +35,19 @@ function config (req, res, next) {
       { name: 'Local', path: '/mapfilter/filters/local', value: 'local' }
     ],
     bingProxy: '/bing-proxy',
-    bingMetadata: '/bing-metadata'
+    bingMetadata: '/bing-metadata',
+    tiles: {
```

Comment:
This information gets send when json/mapfilter-config.json is called.

Always sending it by default since it will just error out if not found, but we could also check for tile existence here and not append this block if no tiles are on disk.